### PR TITLE
[Docenhancement][Cardsbehaviour] [4166]

### PR DESCRIPTION
**Pull request by v-vidhyarani**
common feature of all three cards are added a note.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c4c6044302338a03b4604166a95929676b412699/msteams-platform/task-modules-and-cards/cards/cards-reference.md) | 88 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/330f8105-ec73-42b4-b96f-4f92e744e8e2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approved

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ You can identify and use different types of cards based on your application requ
 
 You can go through some common properties that are applicable to all cards.
 
+> [!NOTE]
+> Hero and thumbnail cards with multiple actions are automatically split into multiple cards in carousel layout.
```
Comment (surbhigupta12) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:75]: Please add article "a"

In a carousel layout

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fc1483328c218928ad7895b2a078c3cc9cd0b35f/msteams-platform/task-modules-and-cards/cards/cards-reference.md) | 88 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/380528ec-401a-4c75-8b7b-e96baba8c327) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**